core/src/edu/asu/c3simulator/screens/TaskManagementScreen.java

```diff
@@ -29,31 +30,17 @@ public TaskManagementScreen(Game game)
 		TaskManagement taskManagementComponents = new TaskManagement(stage, skin);
 		
 		// TODO add Corner Advisor
-		// TODO add Home Button
+		HomeButton home = new HomeButton(game);
 		
-		NavigationPanel navigationPanel = createNavigationPanel();
-		
-		stage.addActor(navigationPanel);
+		stage.addActor(home);
 		stage.addActor(taskManagementComponents);
 	}
 	
 	@Override
-	public NavigationPanel createNavigationPanel()
+	public void createNavigationPanel()
 	{
-		// TODO add screens
-		NavigationPanel navigationPanel = new NavigationPanel(game, skin);
-		navigationPanel.addButton("Company", null);
-		navigationPanel.addSubButton("Company", "Business", null);
-		navigationPanel.addSubButton("Company", "Assets", null);
-		navigationPanel.addButton("Tasks", null);
-		navigationPanel.addSubButton("Tasks", "Manage", null);
-		navigationPanel.addSubButton("Tasks", "Completed", null);
-		navigationPanel.showSubButtonsFor("Tasks");
-		
-		navigationPanel.setPosition(0.01f * stage.getWidth(), stage.getHeight()
-				- (0.3f * stage.getHeight()));
-		
-		return navigationPanel;
+		@SuppressWarnings(value = { "unused" })
+		CouncilPanel navigation = new CouncilPanel(game, stage);
```

Comment:
It was Colonel Mustard, in the bitstream, with the E-clipse. 

Mystery solved. We can all go home.